storybook/storybook-react/src/MegaMenu/MegaMenu.stories.tsx

```diff
@@ -0,0 +1,289 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Grid, Heading, Link, MegaMenu, UnorderedList } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Navigation/Mega Menu',
+  component: MegaMenu,
+  parameters: {
+    layout: 'fullscreen',
+  },
+} satisfies Meta<typeof MegaMenu>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <Grid key="nav" paddingVertical="medium">
```

Comment:
We don't need this `key`, same for the other story

Reply:
Removed

Reply:
Also added a ListWrapper like discussed